src/Apps/Order/Components/OfferInput.tsx

```diff
@@ -20,7 +18,6 @@ export class OfferInput extends React.Component<OfferInputProps> {
         title="Your offer"
         type="text"
         pattern="[0-9]"
-        innerRef={this.inputRef}
```

Comment:
We are actively using innerRef in Positron right now, curious why this needs to be removed?  Is it incompatible with Native?

Reply:
Not sure why it was removed, but do know that it's incompatible with styled-components 4 -- can just use `ref`

Reply:
@eessex damn, we were pretty sure this wasn't used, but didn't CC you - argh!! We removed it mostly because we couldn't find anything using it, clearly we were wrong. I'll reach out on Slack and we can figure out what to do - sorry about that!! ❤️

Reply:
Note that this _will_ have to go away soon once I complete upgrades, so if there's a way to avoid using it that would be good.

Reply:
@damassi looks like the implementation doesn't go away in `styled-components@4`, it is just renamed to `ref`.